ddtrace/contrib/aiohttp/patch.py

```diff
@@ -14,7 +17,10 @@
     template_module = True
 except ImportError:
     template_module = False
-
+except Exception as e:
+    e_str = str(e)
+    log.warning("failed import: aiohttp_jinja2. aiohttp_jinja2 can not be instrumented. %s", e_str, exc_info=True)
```

Comment:
```suggestion
except Exception:
    log.warning("failed import: aiohttp_jinja2. aiohttp_jinja2 can not be instrumented.", exc_info=True)
```

`exc_info=True` should be adding the exception information for us automatically.